plugins/check_users.c

```diff
@@ -115,6 +120,11 @@ main (int argc, char **argv)
 
 	WTSFreeMemory(wtsinfo);
 #elif HAVE_UTMPX_H
+#ifdef HAVE_LIBSYSTEMD
```

Comment:
is systemd somehow dependent on `UTMPX`?

Reply:
It does not depend on UTMP, but all other projects preferred to have utmp support as fallback for the case, that the application is linked against libsystemd, but systemd-logind is not running.
I can change it, that, if compiled against libsystemd, utmp will be always ignored, if wanted.

Reply:
@RincewindsHat should I change the PR to use logind only if compiled against libsystemd and no fallback to utmp, or should I let it as it is and resolve the conversation?

Reply:
I'd prefer it if applications did the same thing here - just fallback.

Reply:
So sorry for the delay, totally on the side of having a fallback, just wondering why this is in the branch of `HAVE_UTMPX_H`.

Reply:
I changed that now.